deploy/olm-catalog/ibm-block-csi-operator-community/1.0.0/csi.ibm.com_ibmblockcsis_crd.yaml

```diff
@@ -2,6 +2,13 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: ibmblockcsis.csi.ibm.com
+  labels:
```

Comment:
we are not supposed to change released bundles for [non cosmetic reasons](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#updating-your-existing-operator).
could we undo these type of changes?

Reply:
I didn't add the labels for 1.0.0-1.2.0 just for a cosmetic reasons , there were lint issues also on these folders/files

Reply:
this just proves my point that we are not supposed to make those changes here.
anyway, why should we care about current lint issues with yamls that we already released?

Reply:
because next time most probably we will get same lint errors on the old versions

Reply:
why should we continue to lint the old versions? can't we test just the latest?

Reply:
the labels for old version have been removed as discussed